# **Current Marine Turtle Projects** {.unnumbered}
## **Kemp's Ridley Project** ##
This is part of an ongoing collaboration with The National Park Service (NPS) to incorporate genetics approaches to improving the accuracy of annual population monitoring and assessment of the Kemp’s ridley  nesting on South Padre Island in Texas. Identification of Nesting Females responsible for nests laid on South Padre Island in Texas. Our goal is to use mtDNA sequencing, in combination with microsatellites and kinship analysis to match nesting females to nests in which the female is not known.  This allows improved accuracy of annual population census, since nests are typically laid by unobserved females.  Our analysis also informs Life history, ecology and mating behavior (e.g. multiple paternity).

## **Leatherback Population Vital Rates: Age to Maturity (GTseq)** ##
Determination of sea turtle population vital rates such as age to maturity, survival, sex ratios, and population size has been an ongoing SAIP priority for NMFS, and particularly for leatherbacks, where lack of basic demographic information continues to limit capacity to conduct more rigorous population and risk assessments. Since 2009, mass sampling of hatchlings has been carried out each nesting season at Sandy Point National Wildlife Refuge(SPNWR), St. Croix, US Virgin Islands as part of the first (“mark”) phase of an innovative long term CMR study to determine the age at first reproduction for leatherbacks. Over 62,000 hatchlings have been sampled to date. Since natal homing is known to occur in the SPNWR nesting population, in 2016 we initiated the second (“recapture”) phase of the CMR experiment to determine whether any of the “tagged” hatchlings from the first large cohort sampled in 2009 have matured and nested. The Marine Turtle Genetics Group is completing genotyping >4,000 St. Croix leatherback hatchlings from the 2009 and 2010 nesting seasons and will begin processing 2011 hatchling samples to expand the baseline. We are using next-generation sequencing (NGS) in order to genotype the hatchlings and compare them to genotypes of adult first time nesters sampled in recent years to see if any have matured and returned to their natal beach to nest.  In FY21 we expanded our SNP panel with additional SNPs and also incorporated microsatellites into our GT-Seq assays, and these will be further applied in FY22.

## **Leatherback Population Vital Rates: Breeding Sex Ratios, STX (GTseq)** ##
The previously increasing trend in leatherback nesting populations throughout the NW Atlantic has reversed in recent years, causing alarm and raising questions as to the status of this RMU or putative DPS. This is the case in St. Croix, with record low numbers of nesters in the last five consecutive years (Stewart et al. in prep). The 1:1 breeding sex ratios estimates for St. Croix were based on analysis for 2009-2011 (Stewart & Dutton 2014), and given the dramatic decline in the last few years, we started to re-evaluate breeding sex ratios in 2019 after a 10-year hiatus to determine if there has been a change in the proportion of males.  This is a unique opportunity to extend a representative evaluation of current breeding sex ratios beyond a decade from the baseline we established in 2009, at what appears to be a period of transition in population trajectory. Analysis of hatchling samples from 2017, 2018, 2019 is being done with FY20-21 funding.  This work is being extended with FY22-23 awarded funding to analyze samples from 2020, 2021 and 2022 to determine breeding sex ratios and provide a 13 year time series for an index northern Caribbean population (St. Croix).

##	**Leatherback Population Vital Rates: Breeding Sex Ratios, Mexico(GTseq)** ## 
This project is postponed from the FY20-21 SAIP funding cycle.  We completed Year 1, but had to postpone YR2/3 which we covered with the GTSeek contract.  It is actually not a deliverable with the FY22-23 Award (we assumed we would catch up), instead the connectivity elements are in the current deliverables (see 3a & b below) 

## **Leatherback Nesting Population Structure in the E. Pacific (GTseq)** ## 
Leatherback nesting females were collected during January - April 2017 (and other years) at an Index monitoring beach (Barra de la Cruz, Oaxaca) as part of a population genetics assessment for Pacific leatherbacks. This project will have many genetic markers (mtDNA, SNPs, microsatellites) utilized to answer a variety of questions pertinent to ongoing management and recovery goals for this Critically Endangered population (population stock structure, breeding sex ratios, multiple paternity). This project is in collaboration with our Mexican partners, Kutzari and CONANP. In FY22-23 samples from nesting sites in Costa Rica and Ecuador samples will be incorporated to generate genomic (SNP/GT-Seq) data for stock structure analysis

##	**Connectivity and relatedness among Eastern Pacific leatherbacks- NGS (GTseq)** ## 
This project applies kinship approaches to determine pairwise relatedness between individuals and individual sampling locations of leatherbacks on nesting beaches (Mexico, Costa Rica), in-water at foraging areas, fishery bycatch and as strandings.  We will genotype samples (GT-Seq SNP) already available at SWFSC from Ecuador, Peru and Chile (n=280), and continue to collect new samples through ongoing collaborations.

##	**Connectivity and relatedness among Northern Caribbean leatherbacks- NGS (GTseq)** ## 
This project applies kinship approaches to determine pairwise relatedness between individuals and individual sampling locations of leatherbacks to determine fine scale demographic structure among nesting beaches across the northern Caribbean. Samples (n=241 embryos, hatchlings and nesters) have already been collected from Puerto Rico, St. Thomas, St. Kitts, and St. Croix (beaches outside of SPNWR) and additional samples will be collected during 2022 and 2023.

##	 **Northern Caribbean leatherback Demographics** ## 
Freshly laid yolkless (“spacer”) eggshells have been collected from unobserved nests at Sandy Point and other beaches around St. Croix and in the wider region (Puerto Rico, Vieques, St. Thomas, St. John) in order to provide a source of DNA to identify the mother. Genetic profiles from these eggshells will be compared to the Sandy Point nester genetic database to find matches and then these data will be used to improve the nester count and identify beach use and nesting demographics by Sandy Point females that lay nests outside the nightly monitored index beach.

##	 **Hawaii Hawksbill Demographics (GTseq)** ## 
Our goal is to use NGS, specifically ddRAD for SNP discovery, followed by GTseq to SNP genotype hatchlings and nesting females for the purpose of assessing breeding sex ratios and improve population census for the Hawaii nesting population.  This will improve demographic information and the quality of population assessments that are Recovery Plan priorities for Pacific hawksbills, and is part of a joint project with the PIFSC and PIRO to implement NOAA’s Sea Turtle Stock Assessment Improvement Plan (SAIP). We will build on prior years work for samples from 2017-2020 nesting seasons by analyzing samples from the 2021 and 2022 seasons to achieve a 6 year dataset. We’ll also start live hatchling sampling for genetic tagging. Objectives are: 
1)	Identify the females responsible for laying unobserved nests on Hawai‘i to improve nesting population counts and reproductive parameters (e.g. numbers nests/females, nesting periodicity, average clutch size, etc.).
2)	Apply kinship approaches to reconstruct parental genotypes from mother-offspring comparison to census males, determine operational sex ratios (OSR; Hays et al 2010) and mating system of the Hawaiian breeding population.

##	**Connectivity and relatedness among Pacific Island hawksbills- NGS (GTseq)** ## 
This project applies kinship approaches to determine pairwise relatedness between individuals (nesters, strandings, foragers, juveniles) to build spatially explicit maps, determine spatial nesting range of individual turtles (e.g. connectivity between nesting beaches), and origin of juveniles and adults foraging around Hawai‘i, as well as extent of dispersal to other Pacific Island foraging areas. We have been awarded FY22-23 funding (S&T STIFA/SAIP). We will build on the current baseline dataset from the most extensively sampled nesting site in Hawai’i to determine demographic structure of this index population and expand sampling to incorporate individual nesting samples from other sites around the Hawaiian Islands (e.g. Maui, Moloka‘i), to map connectivity between nesting sites.  Next, all other samples (in water, strandings, males, juveniles, post-hatchlings) will be incorporated to determine demographic connectivity and origin of adults and juveniles.  We will add samples already obtained from rare nesters on other Pacific Islands to determine connectivity (if any) of individuals in Hawai‘i with breeding populations elsewhere in the Pacific.  Finally, we will genotype samples we have collected from foraging grounds in Guam, Palau, Northern Mariana Islands to determine whether they are from Hawai’i.  Previous mtDNA analysis of these samples detected Hawaii nesting haplotypes that are common at the Guam and Mariana foraging grounds ~6,000 km away, but mtDNA tools are insufficient to confirm this connectivity.  Our pairwise relatedness analysis between all the sampled nesters and foraging turtles will be able to determine whether Hawai’i juveniles disperse to these distant foraging grounds.

## **Pacific Hawksbill population structure- whole mitogenome sequencing** ## 
Add resolution to haplotype EiIP33, which is very common and widespread across the entire Pacific Ocean.

##	**Pacific Hawksbill population structure (standard mtDNA)** ##
This project seeks to advance knowledge of hawksbill turtle rookery stock structure in order to define units to conserve (ESA Recovery Plans) and fill in gaps in the rookery baseline in order to conduct more accurate Mixed Stock Analysis broadly across the Pacific.  This project uses mtDNA sequencing of the standard 816bp control region fragment and will be included in a regional manuscript.

##	**Pacific green turtles nesting and foraging (Fiji)** ## 
The overall goal of this research project is to fill in gaps remaining of stock composition and connectivity of green turtle foraging grounds to ultimately update a Many-to-Many Mixed Stock Analysis on these foraging populations throughout the Pacific.  This also involves adding rookeries to the baseline dataset. The current work in progress involves analyzing mtDNA of samples obtained from various locations in Fiji. Partners in Fiji (funded by PIRO) have managed to find and sample nesting green turtles at one remote location.  These samples (Fiji nesters) need to be sent to SWFSC and will be an analysis priority 1. 

##	**East Pacific green turtles nesting and foraging (SDBay and SG River, SoCal)** ## 
Ongoing foraging population assessment in California Current, Southern California index sites. 

##	**East Pacific green turtles: Baja (Mexico) foraging** ## 
Old samples from extensive study of Baja foraging grounds that were never sequenced with longer primers  

## 	**Florida Bay loggerhead foraging population assessment** ## 
The goal of this project is to determine stock composition (MU & DPS) of loggerheads in Florida Bay. The work in progress involves analyzing mtDNA of samples obtained across multiple years.

## **Atlantic- Gulf of Mexico green turtle stock structure analysis** ## 
This project seeks to advance knowledge of green turtle rookery stock structure in order to define units to conserve (ESA Recovery Plans) and fill in gaps in the rookery baseline in order to conduct more accurate Mixed Stock Analysis broadly across the Caribbean and GOM.  This addresses a current management need to assess critical habitat (ESA:being led by OPR) and estimate the contribution (if any) of rookeries that belong in the S Atlantic DPS to foraging areas in US waters (BiOP: being led by SERO).  This project uses mtDNA sequencing of a suite of established mtDNA markers that are informative at the population level, including the standard 816bp control region fragment, new mitogenomic markers (Shamblin), and a mtDNA SSTR region.

## **Atlantic - N. Caribbean green turtle stock structure analysis** ##
Sandy Point, St. Croix. This is a stand-alone project to complete a comprehensive survey of all nesting sites around St. Croix.  East End beaches and Buck Island were already done (Shultz et al. In press).  Sandy Point now remains to be done.  There is a need to obtain samples from Puerto Rico to resolve DPS uncertainty and we are working with partners to initiate sampling effort in 2022-23.  

## **Establishing life history parameters for Pacific olive ridley turtles** ##
This is part of a multi-faceted project combining skeletochronology, stable isotope and genetic analysis.  We will sequence mtDNA of tissue and/or bone samples from approx 50 olive ridleys strandings to determine whether they belong to the East or West Pacific breeding populations.  This is being funded by FY22 S&T STIFA (SAIP), so labwork needs to be completed in FY22 

## **mtDNA analysis of rare olive ridley nesting in Hawaii** ##
Rare nesting, so there’s interest from PIRO on where this fits with current population structure in Pacific.  We will sequence mtDNA and analyze haplotype(s). 

## **DNA Extraction of hawksbill turtle carapace and products (Shellbank)** ##
The World Wide Fund for Nature (WWF) Australia and Southwest Fisheries Science Center Marine Turtle Genetics Program (MTGP) are collaborating in Australia’s multi-faceted and partnered hawksbill turtle conservation program being delivered in Southeast Asia and Pacific (Asia-Pacific) to address unsustainable use and illegal trade in marine turtles and turtle products. With a focus on hawksbill turtles, one of WWF’s program objectives is to develop and apply new innovative approaches to effectively track marine turtles and turtle products along the trade chain from source to sale. To catalyze enforcement and aid conservation, traceability of hawksbill turtle stocks targeted in trade will require a better understanding of hawksbill turtle genetic origins and relationships between nesting and foraging populations.
The MTGP has the lead responsibility for generating, analyzing and interpreting genetic data by developing the capacity to identify the stock origin of wild and confiscated (illegally trafficked) hawksbill turtle parts and products. The first phase of this project was completed (determining whether DNA can be extracted and sequenced from confiscated “tortoiseshell” jewelry), and now efforts are underway (WWF- Shellbank Project) to develop and apply a “toolkit” for forensics application.   We have samples on hand at SWFSC and propose to continue with extractions and collaboration in FY23 as resources allow.

## **eDNA pilot- SoCal green turtles** ##
Ongoing foraging population assessment in California Current, Southern California index sites.  Collect samples for proof of concept

## **eDNA pilot- leatherbacks** ##
Ongoing foraging population assessment in California Current, collect samples for proof-of-concept potentially for BOEM, etc project Phase 2 with MTEAP.

## **Sea Life Park (Hawaii) captive green turtle kinship** ##
There are 14 captive green turtles at SLP with incomplete histories and an interest in improving husbandry by separating first order relatives that might breed with each other.  We will sequence mtDNA and genotype all samples to conduct kinship and relatedness to determine family relationships.\

## **Fisheries Bycatch, Strandings & MTGP Sample Requests** ##
•	**Pacific Fisheries (Hawaii Longline and American Samoa Longline, International)**
This work is a component of our (MTGP) core mission as NOAA’s Center for sea turtle genetics, which is to determine the stock origin of turtles caught in Hawaii and American Samoa longline fisheries. We have developed and apply a combination of mtDNA and nDNA approaches to assign breeding population origin (DPS, MU or DIP) for all species of sea turtle bycatch.  We have optimized a set of microsatellite markers and a baseline dataset for Pacific leatherbacks that allow us to carry out stock assignment of individual turtles (see Assignment testing project below). We are also moving towards being able to also individually assign green turtles with microsatellites. Routine mtDNA sequencing allows us to perform Mixed Stock Analysis (MSA) for all species. This project is in collaboration with PIFSC and PIRO to address management needs driven by Biological Opinions and Species Recovery Planning, and is an on-going effort.\
•	**Atlantic Leatherback and Loggerhead Longline Fisheries By-catch from the SEFSC**
This work is a component of our (MTGP) core mission as NOAA’s Center for sea turtle genetics, which is to determine the stock origin of turtles caught in longline fisheries monitored by SEFSC/SERO. This project is to determine the stock origin of leatherback and loggerhead turtles caught in Atlantic longline fisheries by analyzing microsatellite profiles (leatherbacks) and mtDNA of all sampled turtles. We recently published the results from the analysis of ~400 leatherbacks (Stewart, K.R., LaCasella, E.L., Roden, S.E. et al. 2016. Ecosphere) in collaboration with SEFSC. Samples are sent semi-regularly, we will continue to process additional batches to add to this on-going project.\
•	**West Coast Turtle Strandings**
This project is to determine the stock origin of stranded turtles along the US west coast by analyzing mtDNA sequences and nDNA genotypes of sampled stranded turtles. Although there is no nesting along the US west coast, we do have greens, loggerheads, leatherbacks and olive ridley turtles that forage in the waters along California, Oregon and Washington. This project is an on-going effort, especially with increasing numbers of stranded individuals in the recent past. This work is in collaboration with WCRO and the US West Coast Stranding Network.  Information serves to gain insight into dispersal, life history and connectivity among Pacific sea turtle populations.\
 
•	**Leatherback Turtle Stock Assignment**
A. Along with our established baseline dataset of globally distributed leatherback turtle populations that genetically characterizes and distinguishes stocks by using a suite of 16 microsatellite markers, we can generate a genotype for an individual and determine its stock origin through assignment analysis. This is ongoing as MTGP receives samples from different regions, and information is applied to several projects as needed (e.g. Fisheries bycatch, strandings, FG/In-water assessment).  The overall goal is to compile a geo-spatial stock-specific distribution map for each ocean basin to better understand the boundaries of leatherback nesting populations in the marine environment (DPS, MU and DIP).

## **Archiving:** ##

Accessioning priorities as of 8 February 2023 in order:/
1.	Strandings, one offs, mostly complete except labeling: Seattle Aquarium, SeaWorld (including olive blood), Pacific Marine Mammal Center, California Academy of Sciences, SWFSC. Should all be done this week. SeaWorld may take slightly longer. I’m waiting on answers from Lexi to finish that one off. PMMC, CAS, and SWFSC were all accessioned this week (MTGP email coming at end of week). Hawaii long line samples from a 2014 shipment were also completed this week.\
2.	St. Croix. Should be complete this week unless there are questions on the data.\
3.	DNA Accessions. Database error has been fixed, and I can accession DNA again.\
4.	Fiji. Inventory underway. \
5.	Remaining San Diego Bay/Seal Beach samples.\
6.	Continue labeling NPAIS whenever possible.\

**Current projects as of 22 March 2022:**
1.	St. Croix 2021 samples and data received. Need inventory, accessioning, and labeling. [15 June: Done.]\
2.	2021 Hawaiian hawksbill hatchling samples and data received. ½ inventoried. Need to finish inventory, accession, and label. Should be doable in short time frame. [26 April Update: Accessioned. Needs labeling.]\
3.	Galapagos samples. Awaiting word that they have cleared USFWS before proceeding. [15 June Update: I would like to relabel these, because I got more complete Field ID data, but they are labeled, etc.]\
4.	Mexico samples. Either very recently or not yet received (Erin coordinating.). Will await word that they’ve cleared USFWS before proceeding. [15 June Update: Inventoried. Most of the data is physically on the vials rather than in the provided datasheet. Questions about import and provenience before I can move forward.]\
5.	NPS 2021 samples. Accessioned. I’m labeling these as time permits around new accessions. [26 April Update: Done.]\
6.	St. Croix 2020. Waiting on data from Kelly Stewart to finish last few females and spacers (questions on data)  and to work on the hatchlings (data not yet received). [15 June Update: Data in prep for accession of hatchlings. Can probably finish in full within a week, but have a couple of mammal priority batches to finish first.]\
7.	2020 Hawaiian hatchlings. Done. Working with Amy to pick a date to get them into bins in the freezer (multiple person job given height of storage/weight of bins). Need to return cooler to Alex. [15 June Update: Cooler back with Alex in HI.]\
8.	NPS 2020 samples. Accessioned. Labeling ongoing. [15 June Update: Done.]\
9.	I’m expecting additional turtle strandings in April or so for accession to go with Heather Harris’ biotoxin study sample send off. I anticipate these being quick turnaround. [15 June Update: Heather Harris samples sent off 13 June 2022.]\
10.	Have asked Barbara Schroeder for the EVER numbers for the most recent Everglades loan samples, so that I can add them to the database and find the samples in our collection when requested for the annual spotcheck. \
11.	Extraction batches, check in/check out DNA/tissue, loan documentation, etc. as needed.\
*Note:* Turtle labeling is one of the big hurdles with limited lab time, because labeling is the most time-consuming part of the accessioning process (assuming the data is pretty clean), and turtles tend to come in large numbers. I’ve been prioritizing inventory through accession, and labeling large batches when I have time around new specimens in mammals or turtles. This will take some time to catch up on the label backlog when we have more lab access.\
*Note 2:* With Kelly retiring, I’m hoping to focus a lot of time on straightening out the years old Mammals backlog (Turtles has one too, but nobody’s retiring that I know of!) before she goes.

